### PR TITLE
MM-59040 Fix issue with hidden threads view on mobile web

### PR DESCRIPTION
**Pull request by matthewbirtch**
#### Summary
Fixes an issue that caused the threads view to be completely hidden on browsers <768px in width.

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/MM-59040

#### Screenshots
|  before  |  after  |
|----|----|
| image | image |
| image | image |

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/threading/global_threads/global_threads.scss [hunk 1]
```diff
@@ -110,8 +110,6 @@
     }
 
     @media screen and (max-width: 768px) {
-        grid-template-rows: 0 1fr;
```
Comment (matthewbirtch) [inline webapp/channels/src/components/threading/global_threads/global_threads.scss:113]: Not really sure what this was needed for anymore or why I left it in there when I did the overall layout/UI changes.

**Review by yasserfaraazkhan — APPROVED** [review]
verified in mobile view and minimising browser, the threads are visible to the user.

**Comment by marianunez** [comment]
@matthewbirtch Devin is out for the next couple of weeks, swapped reviewers out

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by calebroseland — APPROVED** [review]

**Comment by calebroseland** [comment]
@matthewbirtch @marianunez does this need to be cherry-picked?

**Comment by marianunez** [comment]
> @matthewbirtch @marianunez does this need to be cherry-picked?

It does seem to be a regression to 9.9 so I would say ideally yes to cherry pick back to 9.9. but not sure where it is at with a dot release. @amyblais thoughts?

**Comment by amyblais** [comment]
> It does seem to be a regression to 9.9 so I would say ideally yes to cherry pick back to 9.9. but not sure where it is at with a dot release. @amyblais thoughts?

v9.9.1 is currently rolling out, so we can cherry-pick this to v9.10, and to a v9.9.2 dot release once it gets scheduled.

**Pull request merged**